device/iot-device-samples/send-event/README.md

```diff
@@ -1,8 +1,8 @@
-# Send Event Sample
+# Send Events In Batch Sample
```

Comment:
It looks like you edited the wrong readme. Double check the path of this file

Reply:
yeah :)))